frontend/src/views/account-mgmt/ManageUsersView.vue

```diff
@@ -211,8 +207,9 @@ export default {
       if (!users) return []
       return users.sort((a, b) => {
         // Check for account management role and sort by it, with true values first
-        const roleA = a.role === this.ROLES.ACCOUNT_MANAGEMENT
-        const roleB = b.role === this.ROLES.ACCOUNT_MANAGEMENT
+        // TODO (weskubo-cgi) Revisit this requirement given new Roles Security Matrix
+        const roleA = a.role === this.ROLES.ACCOUNT_MANAGER
```

Comment:
since role is an object, we will need to update this line to a.role?.roleName

Reply:
same for b.role

Reply:
Updated and re-tested.